.travis.yml

```diff
@@ -22,10 +22,6 @@ jobs:
   allow_failures:
     - name: "Python - PIP"
 
-after_failure:
-  - cat logs/run_tests_log.txt
-
 import:
-  - pcdshub/pcds-ci-helpers:travis/shared_configs/setup-env-ui.yml
```

Comment:
Removal of this surprises me - and it still works, so 🤷‍♂️ ?

Reply:
https://github.com/pcdshub/pcds-ci-helpers/blob/7588cd5070b0fc55d00313c6ffa1e010cf8eccd9/travis/shared_configs/standard-python-conda.yml#L14

Reply:
I made the change - brain is at its limits of recall:
https://github.com/pcdshub/pcds-ci-helpers/commit/d40226570c929dca82717806265fbd153ccb3aa4